### ISSUE TITLE
Auth Address not returned in /v2/transaction/{txid}

### ISSUE THREAD

**Issue by Blackglade** [algorand/indexer#1318]
### Subject of the issue

The Indexer API Spec clearly defined `auth-addr` as a potential return value for a transaction when hitting the `/v2/transaction/{txid}` endpoint. See image:

<img width="758" alt="Screenshot 2022-11-07 at 1 41 32 PM" src="https://user-images.githubusercontent.com/6363077/200389191-bd8299f2-9590-4819-abd9-3f02b7a72464.png">

This appears not to be the case. Querying several transactions that include the `sgnr` key in the json blob in my indexer database, none of them returned a response that included the `auth-addr`. Not sure if this key is getting left out purposely by the indexer api. I'll include some tx ids below that have the `sgnr` key, but no `auth-addr` response:
```
FLYTOPHI2FXFIMBHJH5GJSK5XGZWOP5L54IJFWRQBCOXMFLQH2NQ
NFY5Y2DZW45WLSPW3CBUGLZDHVKHSENXRE4G2P5ZZG3LEDQJAASA
XA2W3KCBNG75VS7PPXWQFCPA5JGME337N3SWBBNAEP5R3XJOCBAQ
74OJOUDTAKE5L7NAASCOR3NBHDP2BU22ESH2XBVYZ6G3UMYUJJZQ
```

**Comment by urtho** [comment]
Wow.

It seems that `signedTxnWithAdToTransaction` does not convert the AuthAddr from SignedTxnWithAD to generated.Transaction.

**Comment by winder** [comment]
Closed by #1459
